test/miscellaneous_tests/symbolic_stoichiometry.jl

```diff
@@ -3,7 +3,8 @@ t = default_t()
 
 ### Base Tests ###
 
-let
+# Probably requires us to permit Int64 parameters for this to work. 
+@test_broken let
```

Comment:
Both symbolic stoichiometry tests are broken. These are the ones that maybe you should have a look at, since there is quite a lot going on. I am happy to write new ones, but afraid that if I start changing in these I will inadvertently change something that was intended to be there and I did not realise.

Reply:
Let's leave them broken in this PR and followup in another. I added it to the V14 todo.

Reply:
This is fixed in https://github.com/SciML/Catalyst.jl/pull/797